index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    
+    <header>
+        <section class="menu_icon">
+                <div></div>
+                <div></div>
+                <div></div>
+        </section>
+    </header>   
+    
+    <nav class="main_section">
+        <section class="home_section">
+            
+            <h1 class="heading_up"><br>Hey there.<br> I’m Ishimwe.</h1>
+            <h1 class="heading_down">I’m a software developer</h1>
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.</p>
+        </section>
+        <section class="social_links">
+            <img src="icon/twitter.png" alt=""  class="icon">
+            <img src="icon/Linkedin icon.png" alt=""  class="icon">
+            <img src="icon/medium.png" alt=""  class="icon">
+            <img src="icon/github.png" alt=""  class="icon">
+            <img src="icon/angellist.png" alt=""  class="icon">
+            <br>
+        </section>
+    </nav>
```

Comment:
To show good semantics and to enable visitors to be able to connect to your social icons, kindly use a `<ul> <li> </li></ul>` tag for your social icon like this :

>

        <ul class="social_links">
           <li>  <a href=""#><img src="icon/twitter.png" alt=""  class="icon"> </a></li>
          <li>  <a  href="#"><img src="icon/Linkedin icon.png" alt=""  class="icon"> </a></li>
           <li> <a  href="#">  <img src="icon/medium.png" alt=""  class="icon">  </a></li>
           <li> <a  href="#"> <img src="icon/github.png" alt=""  class="icon"> </a></li>
           <li> <a href="#"> <img src="icon/angellist.png" alt=""  class="icon"> </a></li>
            <br>
         </ul>

Reply:
Oh Yes, thanks a lot !